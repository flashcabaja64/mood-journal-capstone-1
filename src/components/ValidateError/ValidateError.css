.error-message {
  color: red;
  padding: .5% 0 .5% 0;
}