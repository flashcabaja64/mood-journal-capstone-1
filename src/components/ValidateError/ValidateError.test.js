import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ValidateError from './ValidateError'

describe('ValidateError component testing', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
  
    ReactDOM.render(<ValidateError />, div);
  
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('renders UI as expected', () => {
    const tree = renderer
        .create(<ValidateError />)
        .toJSON();
        expect(tree).toMatchSnapshot();
  })
})
