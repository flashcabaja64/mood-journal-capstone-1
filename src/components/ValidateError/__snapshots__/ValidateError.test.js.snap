// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidateError component testing renders UI as expected 1`] = `null`;
